t/e2e.t

```diff
@@ -529,6 +529,41 @@ subtest "slow-start" => sub {
     };
 };
 
+subtest "trasport-parameters" => sub {
+    my $guard = spawn_server();
+
+    subtest "max-udp-payload-size" => sub {
+        sub do_test {
```

Comment:
```suggestion
        my $do_test = sub {
```
IIRC, `sub do_test` defines a global function, even if it is defined within an anonymous subroutine.